huaweicloud/services/ces/resource_huaweicloud_ces_alarmrule.go

```diff
@@ -346,7 +362,7 @@ func resourceAlarmRuleRead(_ context.Context, d *schema.ResourceData, meta inter
 
 	r, err := alarmrule.Get(client, d.Id()).Extract()
 	if err != nil {
-		return common.CheckDeletedDiag(d, err, "alarmrule")
+		return common.CheckDeletedDiag(d, err, "Error retrieving HuaweiCloud CES alarm rule")
```

Comment:
I would suggest keep the original `alarmrule` here, as the DiagErrorf format like ("%s: %s", msg, err)